apps/server/src/lib/driver/microsoft.ts

```
@@ -1,446 +1,1194 @@
+          .select('id,displayName,userPrincipalName,mail')
+          .get();
+
+        let photoUrl = '';
+        try {
+          // Requires separate fetching logic
+        } catch (error: unknown) {
+          console.warn(
+            'Could not fetch user photo:',
+            error instanceof Error ? error.message : 'Unknown error',
+          );
+        }
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)